search-app/docs/api.md

```diff
@@ -1961,7 +1961,12 @@ Examples of sorting by start_date in descending(Most recent data first) and asce
 
 ### <a name="retrieving-concepts-by-concept-id-and-revision-id"></a> Retrieve concept with a given concept-id or concept-id & revision-id
 
-This allows retrieving the metadata for a single concept. This is only supported for collections and granules. If no format is specified the native format of the metadata will be returned.
+This allows retrieving the metadata for a single concept. This is only supported
```

Comment:
I hope not ... I've been doing this in other docs, too -- I'm generating locally right now to double-check.